workspaces/spectacle/src/graphql/schema.ts

```diff
@@ -113,11 +113,9 @@ type Endpoint {
   responses: [HttpResponse!]!
 
   # Contributions which define descriptions
-  # TODO figure out what this currently maps to in the contributions list
   contributions: JSON!
 
   # Is the endpoint removed
-  # TODO figure out how this gets mapped
```

Comment:
I totally meant to check this when i added this todo - apparently this slipped through